src/components/Block/SpaceFormProfile.vue

```diff
@@ -63,6 +63,12 @@ const definition = computed(() => {
         title: 'Discord',
         examples: ['Discord handle']
       },
+      votingPowerSymbol: {
+        type: 'string',
+        maxLength: 6,
+        title: 'Voting Power symbol',
```

Comment:
We should always use sentence case
```suggestion
        title: 'Voting power symbol',
```